README.md

```diff
@@ -11,36 +11,32 @@ itself.
 
 ## Deployment
 It is likely that the Dashboard is already installed on your cluster. Check with the following command:
-```bash
-kubectl get pods --all-namespaces | grep dashboard
+```shell
+$ kubectl get pods --all-namespaces | grep dashboard
 ```
 
 
 If it is missing, you can install the latest stable release by running the following command:
-```bash
-kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
+```shell
+$ kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
 ```
 Note that for the metrics and graphs to be available you need to have [Heapster](https://github.com/kubernetes/heapster/) running in your cluster.
 
 ## Usage
-You may access the UI via the apiserver proxy. Open a browser and navigate to `https://<kubernetes-master>/ui`.
-
-Please note, this works only if the apiserver is setup to allow authentication with username and password. Currently, the setup tool `kubeadm` is not doing so. If username and password is unknown to you then use `kubectl config view` to find it.
-
+The easiest way to access Dashboard is to use kubectl. Run the following command in your desktop environment:
+```shell
+$ kubectl proxy
+```
+kubectl will handle authentication with apiserver and make Dashboard available at [http://localhost:8001/ui](http://localhost:8001/ui)
 
+The UI can _only_ be accessed from the machine where the command is executed. See `kubectl proxy --help` for more options.
 
 ## Alternative Usage
-Alternatively, you may access the UI via the kubectl proxy. This is useful if you have a `kubectl`
-configured with access to the cluster, but lack password credentials for use in a browser:
-
-```bash
-kubectl proxy
-```
+You may access the UI directly via the apiserver proxy. Open a browser and navigate to `httpS://<kubernetes-master>/ui`.
 
-This will make the dashboard available at [http://localhost:8001/ui](http://localhost:8001/ui)
+Please note, this works only if the apiserver is setup to allow authentication with username and password. Currently, the setup tool `kubeadm` is not doing so. See [documentation](http://kubernetes.io/docs/admin/authentication/) if you want to configure it manually.
```

Comment:
'...is set up to allow...' --> setup is a noun, set up is the verb.
'This is not currently the case with the setup tool 'kubeadm'.'